core/src/main/resources/application.conf

```diff
@@ -362,7 +362,7 @@ database {
       driver = "slick.driver.HsqldbDriver$"
       db {
         driver = "org.hsqldb.jdbcDriver"
-        url = "jdbc:hsqldb:mem:${uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
+        url = "jdbc:hsqldb:mem:${uniqueSchema};shutdown=false;hsqldb.tx=mvcc;allow_empty_batch=true"
```

Comment:
ToL: Is this still required for all HSQLDB urls? If so, we may want to start documenting these requirements.